modules/mirror-registry-localhost.adoc

```diff
@@ -22,9 +22,10 @@ Installing the _mirror registry for Red Hat OpenShift_ using the `mirror-registr
 [source,terminal]
 ----
 $ sudo ./mirror-registry install \
-  --quayHostname <host_example_com> \
+  --quayHostname <host_example_com> \ <1>
   --quayRoot <example_directory_name>
 ----
+<1> If port 8443 is already in use, you can use another port by specifying `quayHostname <host_example_com>:<port>` instead. Be sure to use the port number that you specify in the rest of the commands instead of 8443.
```

Comment:
I don't think OMR Installation support customize the port, see known issue https://issues.redhat.com/browse/PROJQUAY-2903